athena-dynamodb/src/test/java/com/amazonaws/athena/connectors/dynamodb/DynamoDBIT.java

```diff
@@ -0,0 +1,169 @@
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.athena.connectors.dynamodb;
+
+import com.amazonaws.athena.connector.integ.IntegrationTestBase;
+import com.amazonaws.services.athena.model.Row;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.annotations.Test;
+import software.amazon.awscdk.core.Stack;
+import software.amazon.awscdk.services.iam.Effect;
+import software.amazon.awscdk.services.iam.PolicyDocument;
+import software.amazon.awscdk.services.iam.PolicyStatement;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Integration-tests for the DynamoDB connector using the Integration-test Suite.
+ */
+public class DynamoDBIT extends IntegrationTestBase {
+    private static final Logger logger = LoggerFactory.getLogger(DynamoDBIT.class);
+    private static final String DATABASE_NAME = "default";
+
+    private final String lambdaFunctionName;
+    private final String tableName;
+    private final DdbTableUtils ddbTableUtils;
+
+    public DynamoDBIT()
+    {
+        lambdaFunctionName = getLambdaFunctionName();
+        tableName = String.format("%s_%s", this.getClass().getSimpleName().toLowerCase(),
+                UUID.randomUUID().toString().replace('-', '_'));
+        ddbTableUtils = new DdbTableUtils(tableName);
+    }
+
+
+    /**
+     * Sets up the access policy for the Lambda connector to multiple connector-specific AWS services (e.g. DynamoDB,
+     * Elasticsearch etc...)
+     * @return A policy document object.
+     */
+    protected PolicyDocument getConnectorAccessPolicy()
+    {
+        return PolicyDocument.Builder.create()
+                .statements(ImmutableList.of(PolicyStatement.Builder.create()
+                        .actions(ImmutableList.of("dynamodb:DescribeTable", "dynamodb:ListSchemas",
+                                "dynamodb:ListTables", "dynamodb:Query", "dynamodb:Scan"))
+                        .resources(ImmutableList.of("*"))
+                        .effect(Effect.ALLOW)
+                        .build()))
+                .build();
+    }
+
+    /**
+     * Sets the environment variables for the Lambda function.
+     */
+    protected void setConnectorEnvironmentVars(final Map environmentVars)
+    {
+        // This is a no-op for this connector.
+    }
+
+    /**
+     * Sets up the DDB Table's CloudFormation stack.
+     * @param stack The current CloudFormation stack.
+     */
+    protected void setupStackData(final Stack stack)
+    {
+        ddbTableUtils.setupTableStack(stack);
+    }
+
+    /**
+     * Insert rows into the newly created DDB table.
+     */
+    protected void setupData()
+    {
+        logger.info("Setting up DB table.");
```

Comment:
Do we log the table name anywhere?

Reply:
It's there when we log the stack template, but I added it to this log message as well.